src/main/java/com/iexec/sms/tee/config/SideComputeInternalConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package com.iexec.sms.tee.config;
+
+import com.iexec.sms.api.config.TeeAppConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.util.unit.DataSize;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Positive;
+
+@Configuration
+public class SideComputeInternalConfiguration {
```

Comment:
```suggestion
public class TeeWorkerInternalConfiguration {
```
?

Reply:
Sure, so we can maybe add some more worker-relative configuration in there later. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/119/commits/a305d6436fcd58eae4590abba97f1d9e560eab5b.